Source/WebCore/testing/Internals.idl

```diff
@@ -521,6 +521,11 @@ typedef (FetchRequest or FetchResponse) FetchObject;
     undefined advanceToNextMisspelling();
 
     sequence<DOMString> userPreferredLanguages();
+
+    // This only overrides the languages inside the WebProcess, not the GPUProcess.
+    // To override the language more globally, please add something like this to
+    // your test instead:
+    // <!-- webkit-test-runner [ language=jp,fr ] -->
     undefined setUserPreferredLanguages(sequence<DOMString> languages);
```

Comment:
Can we completely remove this in favor of the html comment in the first line?

Reply:
Maybe, if the tests don't dynamically need to change between several languages. There is a decent amount of tests using this so I didn't attempt in this patch. You're right though, it is something we should try to do. I'll try and follow up.